Source/Core/OrientedBoundingBox.js

```diff
@@ -419,6 +419,45 @@ define([
         return result;
     };
 
+    var scratchMatrix = new Matrix3();
+    var scratchCartesian = new Cartesian3();
+
+    /**
+     * If the given position is not already within the box, projects the given position onto the box.
+     * @param {Cartesian3} position The position being projected onto this OrientedBoundingBox.
+     * @param {Cartesian3} [result] The object onto which to store the result.
+     * @returns {Cartesian3} A projected version of the inputted position if it was not originally within the OrientedBoundingBox.
+     */
+    OrientedBoundingBox.prototype.clampToBounds = function(position, result) {
+        //>>includeStart('debug', pragmas.debug);
+        Check.typeOf.object('position', position);
+        //>>includeEnd('debug');
+        result = Cartesian3.clone(position, result);
+        // to avoid inverse of a zero matrix (since that doesn't exist).
+        if (this.halfAxes.equals(Matrix3.ZERO)) {
+            result.x = 0.0;
```

Comment:
Wouldn't the result be the center of the box in this case?  This is a box with no dimension, but it still has a center position.